README.md

```diff
@@ -10,9 +10,26 @@ Intrigued? Keep on reading!
 
 ## Run the app via Docker
 
+Before building a Docker image, create `.env` file in `docker` directory with the following env variables:
+- `MOMENTO_AUTH_TOKEN`=<YOUR_AUTH_TOKEN>
+- `MOMENTO_CACHE_NAME`=<YOUR_CACHE_NAME>
+- `WEATHER_API_KEY` this is for weather API. Check out [OpenWeather](https://openweathermap.org/) to get an API key.
+
 Build a Docker image for the app:
+```bash
+cd docker
+docker build --tag laravel-example .
+```
+If you run into the API limits from GitHub, add a GitHub personal access token to your composer configuration:
+```bash
+export COMPOSER_AUTH=<YOUR_GITHUB_ACCESS_TOKEN>
+docker build --tag laravel-example --build-arg COMPOSER_AUTH=$COMPOSER_AUTH .
+```
```

Comment:
Somewhere in here I think we should add a note that the build involves compiling the grpc extension and that it should be expected to take a really long time. I know we were both suspicious of how long it was taking, and I'd like to address that upfront in the instructions.

Reply:
Good call.